src/business-logic/services/inews-story-rank-resolver-implementation.spec.ts

```diff
@@ -0,0 +1,236 @@
+import { InewsStoryRankResolverImplementation } from './inews-story-rank-resolver-implementation'
+import { InewsStoryRankResolver } from '../interfaces/inews-story-rank-resolver'
+import { EntityTestFactory } from '../factories/entity-test-factory'
+import { InewsStory } from '../entities/inews-story'
+import { InewsStoryMetadata } from '../value-objects/inews-story-metadata'
+
+describe(InewsStoryRankResolverImplementation.name, () => {
+  describe(InewsStoryRankResolverImplementation.prototype.getInewsStoryRanks.name, () => {
+    describe('when no stories are cached', () => {
+      it('assigns ranks linear', () => {
+        const testee: InewsStoryRankResolver = createTestee()
+        const inewsStoryIdSequence: readonly [InewsStoryMetadata, InewsStoryMetadata, InewsStoryMetadata] = [
```

Comment:
Might i humbly suggest just doing this type for this whole file? :)

```suggestion
        const inewsStoryIdSequence: readonly InewsStoryMetadata[] = [
```

Reply:
We agreed to keep this for consistency :)

Reply:
You may :D But as we talked about, the size of the array matters in the other tests, so to keep it the typings consistent, this will be as it is.